index.js

```diff
@@ -110,6 +110,7 @@ form.addEventListener('submit', (e) => {
     errorMessage.style.display = 'block';
   } else {
     document.querySelector('.error').style.display = 'none';
+    localStorage.setItem('login', checkInputCase);
     e.target.submit();
   }
-});
+});
```

Comment:
- [ ] You have done great so far, but there are still some requirements you need to meet before moving to the next milestone. :+1: :+1: 

> When the user changes the content of any input field, the data is saved to the local storage.
When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

- [ ] According to the project requirement, you are meant to save the data from all form input fields as the user enters his/her details into the local storage. You can add a `key press` or `change` event to your input fields. Also, retrieve the data if it exists from the local storage once the page loads. I'll suggest you implement this. :+1: 

> You must create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.
- Also remember to save this in one single object eg: `const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
` 

- Video reference of your current implementation:

https://github.com/JoyTaribagshaw/My-Portfolio/assets/107632780/86460bcf-e247-42d1-8ec6-c34322e21602


- - You seem to just save the email when it's valid but that's not what was required, this milestone is different from email validation, and user data should be saved before the user clicks on the submit button whether it's valid or not. :smile:

Reply:
Thank you so much for your review @V-Blaze , I have made the changes required.